base/exports.jl

```diff
@@ -950,6 +950,7 @@ export
     sizeof,
 
 # tasks and conditions
+    channeled_tasks,
```

Comment:
I don't think starting N tasks with N channels is the common case. It would be simpler to have more of a drop-in replacement for reading values from a single task like produce and consume do now.